friends/src/App.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import './App.css';
+import FriendList from './Component/FriendList';
+import axios from 'axios';
+
+class App extends Component {
+  constructor() {
+    super()
+
+    this.state = {
+      friends: [],
+      id: '',
+      name: '',
+      age: '',
+      email: '',
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get(`http://localhost:5000/friends`)
+      .then(response => {
+        console.log('This is the axios');
+        console.log(response.data);
+        this.setState({ friends: response.data })
+      })
+      .catch(console.log('error!'))
+  }
+
+  name = (event) => {
+    const newName = event.target.value;
+    this.setState({
+      name: newName
+    })
+  }
+
+  age = (event) => {
+    const newAge = event.target.value;
+    this.setState({
+      age: newAge
+    })
+  }
+
+  email = (event) => {
```

Comment:
let me know if you need any help combining these methods into 1 👂 

Also, try to come up with some better / more helpful method names